shell/src/main/java/org/apache/accumulo/shell/commands/CloneTableCommand.java

```diff
@@ -99,6 +105,8 @@ public Options getOptions() {
     noFlushOption =
         new Option("nf", "noFlush", false, "do not flush table data in memory before cloning.");
     o.addOption(noFlushOption);
+    keepOfflineOption =
+        new Option("o", "offline", false, "do not bring the table online after cloning.");
```

Comment:
keepOfflineOption wasn't added to 'o' so it won't show up on the shell command line as an option.

Reply:
doh